src/docs/components/DocsActiveBlock.jsx

```diff
@@ -22,7 +22,7 @@ class DocsActiveBlock extends Component {
         .reduce((result, key) => {
           result[key] = componentProps[key];
           return result;
-        }, {});
+        }, {key: 'component'});
```

Comment:
To stop React from complaining I have to add a `key` to component, contentBefore, contentAfter